mls-rs/src/group/mod.rs

```diff
@@ -1434,11 +1438,8 @@ where
     ///
     /// This function is used to provide the current group tree to new members
     /// when the `ratchet_tree_extension` is not used according to [`MlsRules::commit_options`].
-    pub fn export_tree(&self) -> Result<Vec<u8>, MlsError> {
-        self.current_epoch_tree()
-            .export_node_data()
-            .mls_encode_to_vec()
-            .map_err(Into::into)
+    pub fn export_tree(&self) -> ExportedTree {
+        ExportedTree::new(self.current_epoch_tree().export_node_data())
     }
```

Comment:
This was already there before, but is cloning the tree necessary here? Exported `ExportedTree` could have a borrow of the nodes, while imported `ExportedTree` could have a copy (can be different types).

Reply:
An alternative to having two types is to define `ExportedTree` as follows:
```rust
struct ExportedTree<'a>(Cow<'a, NodeVec>);
```
The owned case is then `ExportedTree<'static>`.